backend/src/main/kotlin/app/ehrenamtskarte/backend/EntryPoint.kt

```diff
@@ -1,58 +1,41 @@
+import app.ehrenamtskarte.backend.common.webservice.GraphQLHandler
 import app.ehrenamtskarte.backend.common.webservice.WebService
 import app.ehrenamtskarte.backend.stores.importer.freinet.FreinetDataImporter
 import app.ehrenamtskarte.backend.stores.importer.lbe.LbeDataImporter
-import app.ehrenamtskarte.backend.stores.webservice.StoresGraphQLHandler
-import app.ehrenamtskarte.backend.verification.webservice.VerificationGraphQLHandler
 import com.expediagroup.graphql.extensions.print
 import com.github.ajalt.clikt.core.CliktCommand
 import com.github.ajalt.clikt.parameters.options.flag
 import com.github.ajalt.clikt.parameters.options.option
-import graphql.schema.GraphQLSchema
 import java.io.File
 
 
 class Entry : CliktCommand() {
-    private val exportStoresApi by option(help = "Export GraphQL schema of stores component.")
-    private val exportVerificationApi by option(help = "Export GraphQL schema of verification component.")
+    private val exportApi by option(help = "Export GraphQL schema. Given ")
     private val importFreinetData by option("--importFreinet").flag()
     private val importLbeData by option("--importLbe").flag()
 
     override fun run() {
-        val didExportSchema = maybeExportGraphQLSchemas()
-        if (didExportSchema) return
-
-        Database.setup()
-        maybeImportStoreData()
-        WebService().start()
-    }
-
-    private fun maybeExportGraphQLSchemas(): Boolean =
-        maybeExportGraphQLSchema(exportStoresApi, StoresGraphQLHandler.graphQLSchema, "stores").or(
-        maybeExportGraphQLSchema(exportVerificationApi, VerificationGraphQLHandler.graphQLSchema, "verification"))
-
-    private fun maybeExportGraphQLSchema(fileName: String?, schema: GraphQLSchema, apiName: String = "a"): Boolean {
-        if (fileName == null) return false
-        println("Exporting $apiName GraphQL API to $fileName")
-        exportGraphQLSchema(fileName, schema)
-        return true
-    }
-
-    private fun exportGraphQLSchema(fileName: String, schema: GraphQLSchema) = File(fileName).writeText(schema.print())
-
-    private fun maybeImportStoreData() {
-        if (importFreinetData) {
-            println("Importing data from Freinet...")
-            FreinetDataImporter.import()
-        }
-        if (importLbeData) {
-            println("Importing data from Lbe")
-            LbeDataImporter.import()
+        val exportApi = exportApi
+        if (exportApi != null) {
+            val schema = GraphQLHandler().graphQLSchema.print()
+            val file = File(exportApi)
+            file.writeText(schema)
+        } else {
+            Database.setup()
+            if (importFreinetData) {
+                println("Importing data from Freinet...")
+                FreinetDataImporter.import()
+            }
+            if (importLbeData) {
+                println("Importing data from Lbe")
+                LbeDataImporter.import()
+            }
+            WebService().start()
```

Comment:
Did you find the previous single-function code more readable, or was it just easier to undo all the changes at once?

Reply:
To be honest, I think I just reverted the commit :see_no_evil: Should I re-revert it ?

Reply:
I found the all-in-one main function pretty hard to read, but as it would be extra effort to change it again, I don't mind if we just keep it like that.

Reply:
As the function still is quite short, I think it's accetable :)